modules/oe_content_sub_entity/modules/oe_content_sub_entity_document_reference/oe_content_sub_entity_document_reference.post_update.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Content Sub Entity Document reference module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Hide untranslatable fields for all bundles.
```

Comment:
Add a more explicative comment on why we need this, with a link to the drupal.org issue.

Reply:
:+1: